### ISSUE TITLE
Failing to dedent closing docstring triple-quotes

### ISSUE THREAD

**Issue by cas--** [psf/black#545]
I have a function that due to copy-pasting has an extra level of indentation, Black fixes this however it is failing to dedent the closing triple-quote of multi-line docstrings:

```
def func():
        """A multi-line docstring.
        
        Does nothing.
        """
        pass
```
results in this:
```diff
 def func():
-        """A multi-line docstring.
+    """A multi-line docstring.

         Does nothing.
         """
-        pass
+    pass
```

Likely #144 plays a part here with regards the docstring text indentation.

Operating system: Linux
Python version: 3.6
*Black* version: 18.9b0
Does also happen on master: yes

**Comment by JelleZijlstra** [comment]
This is because making this change would  change the contents of the string, and Black avoids doing that. We could plausibly relax that restriction for docstrings, though, which could be covered by #144.

**Comment by minusf** [comment]
i have different case for this:

white:
```
    return json.loads("""
        {
          ...
        }
    """)
```

black:
```
    return json.loads(
        """
        {
          ...
        }
    """
    )
```

But it's easy to fix manually and once I get used to this style, the corner case is gone...

**Comment by ambv** [comment]
This will we solved as part of #144.
